superduperdb/backends/ibis/query.py

```diff
@@ -317,6 +315,24 @@ def _execute(self, parent, method='encode'):
             schema=self._get_schema(),
         )
 
+    @staticmethod
+    def _refactor_insert_data_info(data):
+        merge_data = {}
+        for key in ['_leaves', '_blobs', '_files', '_schema']:
+            if key in data:
+                merge_data[key] = data.pop(key)
+
+        merge_data = base64.b64encode(pickle.dumps(merge_data)).decode()
+        data['_info'] = merge_data
+        return data
+
+    @staticmethod
+    def _recover_insert_data_info(data):
+        if '_info' in data:
+            merge_data = pickle.loads(base64.b64decode(data.pop('_info')))
+            data.update(merge_data)
+        return data
+
```

Comment:
TT Save the necessary information for subsequent decoding, otherwise losing it will result in the inability to recover the artifact/file.

Reply:
Insert data info?

Reply:
Yes, we previously discarded it, which will result in the artifact type data being unable to be decoded.

Reply:
Removed